{"name": "test_b", "status": "failed", "statusDetails": {"message": "AssertionError: assert (1 + 1) == 3", "trace": "self = <DemoPyTest.b_folder.da_folder.test_da.TestDa object at 0x0378C0B0>\n\n    @pytest.mark.db\n    def test_b(self):\n>       assert 1 + 1 == 3\nE       assert (1 + 1) == 3\n\nb_folder\\da_folder\\test_da.py:19: AssertionError"}, "start": 1592389159485, "stop": 1592389159485, "uuid": "4eed8b22-6be0-4fcc-95d4-7ffde1d4ad2c", "historyId": "a2f75ca1f48344895d0777c01151be84", "testCaseId": "44433339a88fe4b6a48caaae4d7ce92b", "fullName": "b_folder.da_folder.test_da.TestDa#test_b", "labels": [{"name": "tag", "value": "db"}, {"name": "parentSuite", "value": "b_folder.da_folder"}, {"name": "suite", "value": "test_da"}, {"name": "subSuite", "value": "TestDa"}, {"name": "host", "value": "R10459"}, {"name": "thread", "value": "23444-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "b_folder.da_folder.test_da"}]}